src/theme/typography.ts

```diff
@@ -0,0 +1,79 @@
+const regular = `
+  font-family: Lato, sans-serif;
+  font-size: inherit;
+  font-weight: inherit;
+  font-style: inherit;
+  font-stretch: inherit;
+  line-height: inherit;
+  letter-spacing: inherit;
+`;
```

Comment:
I think you can make these emotion `css` calls and have a base font that others inherit like
```
const base = css`font-family: Late, sans-serif`
....
const subheading = css`
${base}
font-weight: bold`
```
then if end up changing the font it can be changed everywhere